tools/requirements.txt

```diff
@@ -92,7 +92,7 @@ msgpack==1.0.2; python_version >= "3.6" and python_full_version < "3.0.0" and py
 msrest==0.6.21; python_version >= "3.6"
 mypy-extensions==0.4.3; python_version >= "3.6"
 mypy==0.812; python_version >= "3.5"
-oauth2client==4.1.3; python_version >= "3.6"
+google-auth==1.32.1; python_version >= "3.6"
```

Comment:
If you take a look at the comments at the top of this file:

https://github.com/certbot/certbot/blob/7ede5c348731558fdd2aacbbf7796c3fea4fa904/tools/requirements.txt#L1-L2

it is automatically generated by running `tools/pinning/pin.sh`.

I believe you just need to run that script rather than manually modifying this file. I am not 100% on this so I am doubling checking that advice ^^.

Reply:
I ran but there was no changes in the file